packages/web/config/chain-infos.ts

```diff
@@ -2016,6 +2016,29 @@ const chainInfos = (
       features: ["stargate", "ibc-transfer", "no-legacy-stdTx", "ibc-go"],
       explorerUrlToTx: "https://rebus.explorers.guru/transaction/{txHash}",
     },
+    {
+      rpc: "https://rpc.mainnet.teritori.com/",
+      rest: "https://rest.mainnet.teritori.com/",
+      chainId: "teritori-1",
+      chainName: "Teritori",
+      bip44: {
+        coinType: 118,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("tori"),
+      currencies: [
+        {
+          coinDenom: "TORI",
+          coinMinimalDenom: "utori",
+          coinDecimals: 6,
+          coinGeckoId: "pool:utori",
+          coinImageUrl: "/tokens/tori.png",
+          isStakeCurrency: true,
+          isFeeCurrency: true,
+        },
+      ],
+      features: ["stargate", "ibc-transfer", "no-legacy-stdTx", "ibc-go"],
+      explorerUrlToTx: "https://explorer.teritori.com/teritori/tx/{txHash}",
+    },
```

Comment:
This is not mandatory, but I highly recommend adding your gas price steps (see the chain schema file). If someone already has the chain config previously with the gas prices from elsewhere, this isn't a problem but if you *don't* explicitly specify this, if someone acquires the chain config via Osmosis then you'll end up picking up the default gas price steps rather than your custom ones.